docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+version: '3'
+
+services:
+
+  consul-agent-1: &consul-agent
+    image: consul:latest
+    networks:
+      - trento
+    command: "agent -retry-join consul-server-bootstrap -client 0.0.0.0"
+
+  consul-agent-2:
+    <<: *consul-agent
+
+  consul-agent-3:
+    <<: *consul-agent
+
+  consul-server-1: &consul-server
+    <<: *consul-agent
+    command: "agent -server -retry-join consul-server-bootstrap -client 0.0.0.0"
+
+  consul-server-2:
+    <<: *consul-server
+
+  consul-server-bootstrap:
+    <<: *consul-agent
+    ports:
+      - "8400:8400"
+      - "8500:8500"
+      - "8600:8600"
+      - "8600:8600/udp"
+    command: "agent -server -bootstrap-expect 3 -ui -client 0.0.0.0"
+
+  trento-webserve:
+    image: trento:latest
+    networks:
+      - trento
+    ports:
+      - "8080:8080"
+    expose:
+      - "8080"
```

Comment:
I don't think we need this here, because it's in the Dockerfile already. What does it do?

Reply:
Yes, it's not really needed as we are exposing the port in the Dockerfile for the container image. It's a leftover from a override I have in my setup, as I have port 8080 already used in my dev machine and use this to switch it to 8082.

I'll remove it, thanks!

Reply:
Indeed, it's pointless even in my usecase as I could simply change the port mapping in the Dockerfile :)